recipes/agat/meta.yaml

```diff
@@ -58,6 +58,8 @@ about:
   license_family: GPL
   license_file: LICENSE
   summary: Another Gff Analysis Toolkit (AGAT). Suite of tools to handle gene annotations in any GTF/GFF format.
+  description: "AGAT has the power to check, fix, pad missing information (features/attributes) of any kind of GTF and GFF to create complete, sorted and standardised gff3 format. Over the years it has been enriched by many many tools to perform just about any tasks that is possible related to GTF/GFF format files (sanitizing, conversions, merging, modifying, filtering, FASTA sequence extraction, adding information, etc). Comparing to other methods AGAT is robust to even the most despicable GTF/GFF files."
```

Comment:
Maybe split that description over multiple lines like [this](https://github.com/bioconda/bioconda-recipes/blob/a572f745527c762810bc4b482f076ed17d161cee/recipes/start-asap/meta.yaml#L39)?